density_scaler/src/hex.rs

```diff
@@ -0,0 +1,300 @@
+use h3ron::{FromH3Index, H3Cell, Index};
+use itertools::Itertools;
+use lazy_static::lazy_static;
+use serde::Serialize;
+use std::{cmp, collections::HashMap, ops::Range};
+
+pub struct HexResConfig {
+    pub neighbors: u64,
+    pub target: u64,
+    pub max: u64,
+}
+
+impl HexResConfig {
+    pub fn new(neighbors: u64, target: u64, max: u64) -> Self {
+        Self {
+            neighbors,
+            target,
+            max,
+        }
+    }
+}
+
+type HexMap = HashMap<H3Cell, u64>;
```

Comment:
If HashMaps become a bottleneck, I'm happy to add whatever is needed to make HexTrees usable here.